src/protocol.js

```diff
@@ -65,7 +65,7 @@ export default function protocol({ args, name, fallback }: ProtocolDefinition) {
     const key = getValueKey(id, value);
     const implementation = (key && implementations[key]) || fallback;
     if (!implementation) {
-      throw new Error(`${name}: not implmented for type \`${value}\``);
+      throw new Error(`${name}: not implemented for type \`${value}\``);
```

Comment:
Is this error message part of the public interface (could someone catch this error and write code relying on the improper spelling)? If so, changing this might require a major version bump.

Reply:
Since this change is small enough and shouldn't impact any existing users, a minor version bump is more appropriate here

Reply:
Sadly this is where I feel like semver breaks down (it isn't immediately clear if the public interface includes the text of the error messages).

Since it's a very small change, there are no people depending on the text of the error internally at HS, and it doesn't seem like there's anyone depending on it externally ([at least not in code pushed to GH](https://github.com/search?q=%22implmented%22+transmute&type=Code)), we can probably suffice with a minor version bump.

As Francois mentioned to me separately, this error messages could *probably* be interpreted as internal to the library (I don't think we document it anywhere) and if someone is relying on the particulars of this error message, they've accepted the risk of it breaking in a minor/patch version.